.circleci/config.yml

```diff
@@ -164,6 +165,7 @@ jobs:
       - checkout
       - run: *setup
       - run: *ios-cordova
+      - run: *install-sfdx
```

Comment:
forcehybrid-ios and forcehybrid-android jobs are failing on CI because the hybridlwc app can't be built.